tc

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+# Taskcat wrapper script.
+
+# Annoyingly 'taskcat deploy' doesn't support --exclude-prefix
+# EXCLUDE_OPTIONS="--exclude-prefix --exclude-prefix helm-charts/"
+
+AWS_REGION=${AWS_REGION:-$(aws configure get region)}
+AWS_PROFILE=${AWS_PROFILE:-default}
+
+if [ $# != 2 ]; then
+  >&2 echo "Usage: tc ACTION TESTCASE"
```

Comment:
It's probably worth adding a `--help` to this script - if only to list the supported actions.